specification/resources/resource-manager/Microsoft.Resources/preview/2019-10-01-preview/examples/DeploymentScripts_GetLogsDefaultWithTail.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "api-version": "2019-10-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "script-rg",
+    "scriptName": "MyDeploymentScript",
+    "tail": "5" 
```

Comment:
Type mismatch with the definition in swagger?

Reply:
@raych1 Fixed.